app/views/admin/categories/new.html.erb

```diff
@@ -1,4 +1,15 @@
-<h1>New Topic</h1>
+<div class="container main mt-5">
```

Comment:
1. Change to Spaces: 4 because this is an HTML file.
Do not waste time next time; fix, change and double-check next time your code so that we do not waste time with this same problem.

2. Also my request was to put DELETE function in _header.html.erb because your LOGOUT function is not working. This is already the answer on how to solve your LOGOUT problem.
**DOUBLE CHECK your code next time.**
It is not good practice to just let client check your application and then basic functions like Logout or Create Category is not working.